plugin/db/util/driverutil.go

```diff
@@ -177,28 +174,32 @@ func ExecuteMigration(ctx context.Context, l *zap.Logger, executor MigrationExec
 		if !m.CreateDatabase {
 			_, err := executor.GetDbConnection(ctx, m.Database)
 			if err != nil {
+				endMigration(ctx, l, executor, startedNs, insertedID, "", false /*isDone*/)
```

Comment:
defer function does action to returned values. without it causes redundant call endMigration(). I prefer to defer func. WDYT @d-bytebase

Reply:
SG